webapp/sso.py

```diff
@@ -51,7 +51,9 @@ def logout():
     def before_request():
```

Comment:
This file was created by @tbille, so I guess I'll address this question to both of you:

In our other apps we have an `authorization_required()` decorator that we put above views. Do you know why we don't do this here? This seems to split up the concerns in a way that's unnecessarily surprising.

Reply:
Oh ok now I see - the view is created by the discourse module and so it's hard to wrap the authorization around it... OK I guess this is alright for now. If I get time I'll see if I can think of a better structure.